### DISCUSSION TITLE
Recording in UTC time

### DISCUSSION THREAD

**Discussion by tbnguyen1407** [bluenviron/mediamtx#2942] [Questions]
### Question

Hello, I need help with storing recordings in UTC time. Current structure uses local time instead of UTC.

```
~ mediamtx.yml ~
pathDefaults:
  recordPath: ./recordings/%path/%Y/%m/%d/%H/%Y%m%d_%H%M%S
```

Any config to specify the timezone to the server?

Thanks.

**Comment by aler9** [comment]
Hello, yes, use the `TZ` environment variable:

```
TZ=UTC ./mediamtx
```

**Reply by A-Asadi** [reply]
Hello,
TZ=UTC ./mediamtx
works fine in Linux but how run mediamtx in windows with TZ variable without changing local system time zone?
thanks
